build/org.eclipse.cdt.make.ui/src/org/eclipse/cdt/make/internal/ui/wizards/NewMakefileProjectWizard.java

```diff
@@ -50,7 +50,7 @@ public void createControl(Composite parent) {
 
 				Button genSourceButton = new Button(buttonComp, SWT.CHECK);
 				genSourceButton.setLayoutData(new GridData(SWT.FILL, SWT.CENTER, true, false));
-				genSourceButton.setText("Generate Source and Makefile");
+				genSourceButton.setText("Create Hello World Source and Makefile example"); //$NON-NLS-1$
```

Comment:
This string should be translatable, so until it is lets leave off the non-translated string comment.

```suggestion
				genSourceButton.setText("Create Hello World Source and Makefile example");
```